tutorials/camkes-vm-kernel-module/module_minimal/settings.cmake

```diff
@@ -0,0 +1,18 @@
+<!--
+  Copyright 2021 Michael Neises
+
+  SPDX-License-Identifier: BSD-2-Clause
+-->
```

Comment:
I think what the CMake style checker is choking on is the comment style. This should work:
```suggestion
#
# Copyright 2021 Michael Neises
#
# SPDX-License-Identifier: BSD-2-Clause
#
```